### DISCUSSION TITLE
What is "Dynamic" tag in visualize library in t-pot 22.04?

### DISCUSSION THREAD

**Discussion by husenclip** [telekom-security/tpotce#1096] [Q&A]
I have two questions.
①In the visualization library, it looks like a new Dynamic tag was added in tpot 22.04. What is the functionality of this?

②I have confirmed that there are some names that are very similar with - Dynamic and some without. For example, Attacker Src IP Reputation and Attacker Src IP Reputation - Dynamic.What is the difference between these?

※The dynamic tag refers to the one in the attached image
![dynamic_tag](https://user-images.githubusercontent.com/102123410/168246210-0cb8f758-09c5-4594-8fa3-e12b22ed5009.png)
.

**Comment by t3chn0m4g3** [comment]
In the past each of the visualization objects was referring to just one honeypot (static). Now there are generic objects which use filters from the corresponding dashboards which results in fewer objects; hence the `Dynamic` tag.

**Reply by husenclip** [reply]
Thank you very much.
①the essense of Dynamic tag in Visualize Library is that the query is "query_string":  {"query": "*"}"?

②Does '"query_string": {"query": "*"}' means a "search against the access logs of all the honeypots that make up the T-Pot" and We can format the data for any honeypot by filtering it on the dashboard side?
